gke/variables.tf

```diff
@@ -75,7 +75,7 @@ variable "enable_nat" {
 
 variable "enable_bastion" {
   type        = bool
-  default     = true
+  default     = false
   description = "Include a bastion/jump server in deployment. You can restrict the range of IPs that can connect to the bastion using `allowed_cidr_blocks`"
```

Comment:
Worth calling out that this is deprecated and will be removed shortly?

Reply:
Wouldn't the release notes be the appropriate place for this? Or you think we should also do it here? Or in there `terraform.tfvars.template` file? All three?

Reply:
Hmm, along with release notes, the template file seems to make the most sense to me - as that is probably what a new customer would see.

Reply:
Sounds good. Fixed the failing `fmt` check and added a note on the deprecation of the bastion host setting